plugins/@grouparoo/google-sheets/src/lib/sheet-import/downloadAndRefreshFile.ts

```diff
@@ -20,7 +20,12 @@ export async function downloadAndRefreshFile(
   );
 
   const localDir = path.join(tmpdir(), "google-sheets-cache", sourceId);
-  const localPath = path.join(localDir, `${sheet.docId}.csv`);
+  const fileSuffix = `${sheet.docId}-${sheet.sheetId || "default"}.csv`;
```

Comment:
I'm not sure the `currentCount` is really helping us much here.
What if the current `fileSuffix` was the subdirectory (and we did `mkdirp` later) and then the actual filename was `${guid}.csv`?
then there would be no chance it would conflict and the folder strcuture still makes it possible to clean up a bit.